sdk/imagebuilder/arm-imagebuilder/review/arm-imagebuilder.api.md

```diff
@@ -10,19 +10,6 @@ import { OperationState } from '@azure/core-lro';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { SimplePollerLike } from '@azure/core-lro';
 
-// @public
-export interface CloudError {
-    error?: CloudErrorBody;
-}
-
-// @public
-export interface CloudErrorBody {
-    code?: string;
-    details?: CloudErrorBody[];
-    message?: string;
-    target?: string;
-}
-
```

Comment:
why this removal is not breaking?

Reply:
it is only used in response 
![image](https://github.com/Azure/azure-sdk-for-js/assets/98569699/2e5b74a1-62b3-47bc-a594-8c5f33f855b9)
no models uses this as a parameter, not sure whether this is the reason

Reply:
i prefer to take it as breaking, could you add the wordings like below?

```
Change CloudError  to ErrorResponseXXX
Change CloudErrorBody  to ErrorDetailXXX
```